src/Dependencies/Json.net/Json.net.csproj

```diff
@@ -1,12 +1,6 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <IsPortable>false</IsPortable>
```

Comment:
It was meant to distinguish portable projects but because now all dependency projects are either netstandard or framework, we don't need this anymore.